lib/perl/Genome/InstrumentData/AlignmentResult/Speedseq.pm

```diff
@@ -0,0 +1,101 @@
+package Genome::InstrumentData::AlignmentResult::Speedseq;
+
+use strict;
+use warnings;
+use Genome;
+
+class Genome::InstrumentData::AlignmentResult::Speedseq {
+    is => 'Genome::InstrumentData::AlignmentResult',
+    has => [
+    ],
+};
+
+sub _run_aligner {
+    my $self = shift;
+
+    #Run get_bam_file to fake revivify the per-lane bams
+    $self->_prepare_output_directory;
+    $self->get_bam_file;
+    return 1;
+}
+
+#Use merged bam for header since we don't have an original per-lane bam
+sub source_bam_path_for_header {
+    my $self = shift;
+    return $self->get_merged_bam_to_revivify_per_lane_bam;
+}
+
+#Don't create flagstat during revivification - postprocess_bam_file will take
+#care of that
+sub create_bam_flagstat_and_revivify {
+    my ($self, $merged_bam, $revivified_bam) = @_;
+
+    my $cmd = Genome::InstrumentData::AlignmentResult::Command::RecreatePerLaneBam->create(
```

Comment:
I pushed a commit that will pass this file along to the `comparison_flagstat` param of `RecereatePerLaneBam`, if it exists.